### PR TITLE
added multiple sensitivity_score fields to the google_data_loss_prevention_inspect_template

### PR DESCRIPTION
**Pull request by abheda-crest**
Added multiple `sensitivity_score` fields in the `info_type` block to the `google_data_loss_prevention_inspect_template` resource.

If this PR is for Terraform, I acknowledge that I have:

- [x] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [x] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [x] [Generated Terraform providers](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/generate-providers.md), and ran [`make test` and `make lint`](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/run-provider-tests/#run-unit-tests) in the generated providers to ensure it passes unit and linter tests.
- [x] [Ran](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/run-provider-tests.md) relevant acceptance tests using my own Google Cloud project and credentials (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [x] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
dlp: added multiple `sensitivity_score` field to `google_data_loss_prevention_inspect_template` resource
```

### CODE REVIEW

**Review by shuyama1 — COMMENTED** [review]
Overall looks good! Got one question: does it allow users to unset this field? If so, can we omit some of the `sensitivity_score` blocks in the update tests to test if unsetting the field works as intended. Thanks!

File: mmv1/products/dlp/InspectTemplate.yaml [hunk 1]
```diff
@@ -139,6 +139,19 @@ properties:
                       name: 'version'
                       description: |
                         Version name for this InfoType.
+                    - !ruby/object:Api::Type::NestedObject
+                      name: 'sensitivityScore'
+                      description: |
+                        Optional custom sensitivity for this InfoType. This only applies to data profiling.
+                      properties:
+                        - !ruby/object:Api::Type::Enum
```
[the remark this answers is not in the document]
Comment (abheda-crest) [reply mmv1/products/dlp/InspectTemplate.yaml:166]: Thanks for pointing that out. I've made the `score` subfield required.

**Comment by abheda-crest** [comment]
> Overall looks good! Got one question: does it allow users to unset this field? If so, can we omit some of the `sensitivity_score` blocks in the update tests to test if unsetting the field works as intended. Thanks!

I've omitted some of the `sensitivity_score` blocks in the update tests.

**Review by shuyama1 — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**